src/index.html

```diff
@@ -11,8 +11,36 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__img"></div>
+      <div class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="product-card__code">Product code: 195434</div>
+      <div class="product-card__stars">
```

Comment:
Fix classes and add the modifier classes
```suggestion
      <div class="stars stars--4">
```